tests/unit/fem/test_pa_kernels.cpp

```diff
@@ -0,0 +1,227 @@
+#include "catch.hpp"
```

Comment:
Please add the Copyright header in the beginning of this file (as in the other tests).